### PR TITLE
Enable external delegate for benchmark_model/label_image for tflite

### PR DESCRIPTION
**Pull request by sunshinemyson**

### CODE REVIEW

**Comment by sunshinemyson** [comment]
Hi @terryheo ,

Could you review this patch? Thanks

**Comment by gbaned** [comment]
@terryheo Can you please review this PR ? Thanks!

**Comment by terryheo** [comment]
Looks like the similar PR with https://github.com/tensorflow/tensorflow/pull/52866

**Comment by gbaned** [comment]
@sunshinemyson Can you please resolve conflicts? Thanks!

**Comment by sunshinemyson** [comment]
> @sunshinemyson Can you please resolve conflicts? Thanks!

updated.

File: tensorflow/lite/CMakeLists.txt [hunk 1]
```diff
@@ -387,6 +387,16 @@ else()
     "${TFLITE_SOURCE_DIR}/nnapi/nnapi_implementation_disabled.cc"
   )
 endif()
+if(TFLITE_ENABLE_EXTERNAL_DELEGATE)
```
Comment (terryheo) [inline tensorflow/lite/CMakeLists.txt:390]: Who defines `TFLITE_ENABLE_EXTERNAL_DELEGATE` ?

File: tensorflow/lite/CMakeLists.txt [hunk 2]
```diff
@@ -458,6 +468,7 @@ endif()
 set(TFLITE_INCLUDE_DIRS
   "${TENSORFLOW_SOURCE_DIR}"
   "${TFLITE_FLATBUFFERS_SCHEMA_DIR}"
+  "${TFLITE_DELEGATES_EXTERNAL_INCL}"
```
Comment (terryheo) [inline tensorflow/lite/CMakeLists.txt:471]: Is this needed?

File: tensorflow/lite/CMakeLists.txt [hunk 3]
```diff
@@ -515,6 +527,10 @@ target_link_libraries(tensorflow-lite
     ${TFLITE_TARGET_DEPENDENCIES}
 )
 
+if (ANDROID_TOOLCHAIN)
+    list(APPEND tensorflow-lite log)
```
Comment (terryheo) [inline tensorflow/lite/CMakeLists.txt:531]: why?

**Review by terryheo — COMMENTED** [review]
Thanks for your contribution.

**Comment by gbaned** [comment]
@sunshinemyson Can you please check @terryheo's comments and keep us posted ? Thanks!

**Pull request closed without merging**